src/cbapi/auth.py

```diff
@@ -60,7 +65,7 @@ def __init__(self):
         environ = os.environ
         self.cbapi_url = environ.get('CBAPI_URL',None)
         self.cbapi_token = environ.get('CBAPI_TOKEN',None)
-        self.cbapi_ssl_verify = environ.get('CBAPI_SSL_VERIFY',None)
+        self.cbapi_ssl_verify = environ.get('CBAPI_SSL_VERIFY',True)
```

Comment:
This doesn't work the same as the file credential store - see https://github.com/carbonblack/cbapi-python/blob/master/src/cbapi/auth.py#L46

Reply:
basically this code won't work as written - if you put "FALSE" or anything in `CBAPI_SSL_VERIFY` it will be "true"